templates/rancher.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>rancher</Name>
+  <Repository>rancher/rancher:latest</Repository>
+  <Registry>https://hub.docker.com/r/rancher/rancher</Registry>
+  <Network>bridge</Network>
+  <Privileged>false</Privileged>
+  <Support>https://rancher.com/docs/rancher/v2.x/en/installation/other-installation-methods/single-node-docker/</Support>
+  <Project>https://github.com/rancher/rancher</Project>
+  <Overview>A container management platform built for organizations that deploy containers in production. The docker version of Rancher cannot be made HA (Highly Available), however the clusters it creates can be. It is also possible to connect to and use clusters created by rancher when it is down. Useful as a management interface for deploying, scaling, and editting apps on other systems. (in the cloud, a raspberry pi cluster, vms running k3os, etc.) Perfect for running and managing the apps that you'd rather not go down while maintaining your server.</Overview>
+  <Category/>
```

Comment:
Please add a category
https://selfhosters.net/docker/templating/templating/#21-categories

Reply:
Fixed in ebc43c1d1b4c9a87b65f84a84f4a299fa7394786